app/controllers/invitations_controller.rb

```diff
@@ -1,22 +1,22 @@
 class InvitationsController < Devise::InvitationsController
+  # Copied from https://github.com/scambra/devise_invitable/blob/master/app/controllers/devise/invitations_controller.rb
+  # And edited. This is a common devise pattern, similar to OmniauthCallbacksController.
```

Comment:
Added these back after accidentally removing them during my refactor otherwise, the comments are unchanged.